docs/Reference/API-Methods.md

```diff
@@ -5277,6 +5398,86 @@ or `null` if no receipt found.
     }
     ```
 
+### priv_newFilter
+
+Creates a [log filter](../Concepts/Events-and-Logs.md) for a private contract. To poll for logs associated with the
+created filter, use [priv_getFilterChanges](#priv_getfilterchanges).
```

Comment:
You can use `priv_getFilterChanges` or `priv_getFilterLogs` to retrieve logs.

Reply:
Added (and for eth_newFilter)